### PR TITLE
add min and max to `rescale_to_grid` manually

### LINKED ISSUE #84: `rescale_to_grid` needs min max passed, defaulting to the current behavior

### PR DESCRIPTION
**Pull request by rfl-urbaniak**
Resolves #84 .

### CODE REVIEW

**Comment by rfl-urbaniak** [comment]
- add four datasets for minimal testing
- format/lint/test using the scripts
- run at least two existing notebooks locally to make sure no troubles arise
- animate your data after conversion to ensure proper behavior

**Comment by PalkaPuri** [comment]
@rfl-urbaniak  -- updated rescale_to_grid, confirmed that it did not break existing code, and also ran checks with imported fish data (docs/experimental/fish/fish_import_checks.ipynb). Have a look when you get a chance. I cannot assign you to review though as the PR was opened by you

File: collab/foraging/toolkit/subsampling.py [hunk 1]
```diff
@@ -20,14 +20,21 @@ def subset_frames_evenly_spaced(df_raw, desired_frames=300):
     return df
 
 
-def rescale_to_grid(df, size):
-    def rescale_column(column, size=size):
-        mapped = (column - column.min()) / (column.max() - column.min())
+# updated function to allow user to pass min and max
+def rescale_to_grid(df, size, gridMin=None, gridMax=None):
+    def rescale_column(column, size=size, gridMin=None, gridMax=None):
```
Comment (rfl-urbaniak) [inline collab/foraging/toolkit/subsampling.py:25]: As is, gridMin and gridMax of rescale_to_grid are not used at all, as you reset them in rescale_column.

**Review by rfl-urbaniak — COMMENTED** [review]
Love the experimental notebook, one more thing to fix (the argument inheritance in rescale_to_grid.

File: collab/foraging/toolkit/subsampling.py (hunk 1 above)
Comment (PalkaPuri) [reply collab/foraging/toolkit/subsampling.py:25]: Talked about this in a meeting, no changes needed - ready to merge!

**Pull request merged**